db/schema.rb

```diff
@@ -67,25 +69,25 @@
   add_index "assignments", ["user_id", "task_id"], name: "index_assignments_on_user_id_and_task_id", unique: true, using: :btree
 
   create_table "bank_accounts", force: :cascade do |t|
-    t.string   "name",                                                           null: false
-    t.string   "iban"
-    t.string   "description"
-    t.decimal  "balance",                   precision: 12, scale: 2, default: 0, null: false
+    t.string   "name",                      limit: 255,                                      null: false
```

Comment:
all of these changes were added when i applied `rake db:migrate` - they are a bit concerning.

Reply:
perhaps name 'limit 255' is not as worrying, but see later ones

Reply:
maybe they are not worrying.  but i am confused by why they are different.

Reply:
seams that my `db:migrate` created a different `schema.rb`. i usually use SQLite. so that's nothing you did wrong. you just see now every "wrong" database migration I did over the last years 😀 i should be no problem, but to keep history clean (one commit does only one thing) just remove the lines from the commit and everything should be ok

Reply:
ah!  i think you are correct, because this was generated from running against mysql.

i hate commits that show unrelated changes, so i will regenerate schema.rb with sqllite.  if we want to clean it up and generate against mysql, that should be another PR.